### PR TITLE
Updated home page

### PR DESCRIPTION
**Pull request by Faro0q**
Changed website name and pic
image

### CODE REVIEW

**Comment by Faro0q** [comment]
Added new layout for home page. Also added a weather button

image

image

File: client/public/index.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 <head>
 	<meta charset="utf-8" />
-	<link href="%PUBLIC_URL%/favicon.ico" rel="icon" />
+	<link href="%PUBLIC_URL%/chicken.jpg" rel="icon" />
```
Comment (wcygan) [inline client/public/index.html:5]: Very nice touch

**Review by wcygan — APPROVED** [review]
LGTM

**Comment by Faro0q** [comment]
Added confirmation on logout
image

File: client/public/index.html (hunk 1 above)
Comment (Faro0q) [reply client/public/index.html:5]: 💋

**Comment by wcygan** [comment]
yessss

**Pull request merged**